src/components/product.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../styles/product.css';
+class Product extends React.Component {
```

Comment:
Functional component here too. You should be able to get away with a classless codebase.